analyzers/tests/SonarAnalyzer.UnitTest/MetadataReferences/NuGetMetadataFactory.Package.cs

```diff
@@ -20,20 +20,27 @@
 
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
 using System.IO;
 using System.Linq;
 using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.Configuration;
+using NuGet.Packaging;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using NuGet.Versioning;
 
 namespace SonarAnalyzer.UnitTest.MetadataReferences
 {
     public static partial class NuGetMetadataFactory
     {
         private class Package
         {
-            public string Id { get; }
-            public string Version { get; }
-            public string Runtime { get; }
+            private string Id { get; }
+            private string Version { get; set; }
+            private string Runtime { get; }
```

Comment:
Since these became private, we don't need properties and we can make them fields.